### PR TITLE
fujitsu-ssl2: Add the headers method

### PR DESCRIPTION
**Pull request by ghost**
Add the headers method.

### CODE REVIEW

File: var/spack/repos/builtin/packages/fujitsu-ssl2/package.py [hunk 1]
```diff
@@ -142,3 +142,12 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         env.append_flags(
             "FCC_ENV", "-idirafter " + path
         )
+
+    @property
+    def headers(self):
+        path = join_path(
+            self.spec.prefix, "clang-comp", "lib64", "clang", "7.1.0", "include"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fujitsu-ssl2/package.py]: Is it always `lib64` or will it be `lib` on some systems? Is it always this same version of clang? We could just search recursively from the beginning of the prefix. Or we could use wildcards.

Comment (ghost) [reply var/spack/repos/builtin/packages/fujitsu-ssl2/package.py]: Thank you for your advice.
I fixed the recipe to do not include "version number" and "lib64" in path according to your advice .

File: var/spack/repos/builtin/packages/fujitsu-ssl2/package.py [hunk 2]
```diff
@@ -142,3 +143,12 @@ def setup_dependent_build_environment(self, env, dependent_spec):
...
             "FCC_ENV", "-idirafter " + path
         )
+
+    @property
+    def headers(self):
+        path = join_path(
+            self.spec.prefix, "clang-comp"
+        )
+        headers = find_headers('cssl', path, recursive=True)
+        headers.directories = os.path.dirname(headers[0])
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fujitsu-ssl2/package.py]: Is this line needed, or does the default `directories` attribute work?

Comment (ghost) [reply var/spack/repos/builtin/packages/fujitsu-ssl2/package.py]: Thank you for your advice . 
I removed unnecessary lines according to your advice .

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**